jit/v2/JIT_v2.0.0.yaml

```
@@ -746,28 +1039,86 @@ paths:
         - $ref: '#/components/parameters/universalImportVoyageReferenceQueryParam'
         - $ref: '#/components/parameters/universalExportVoyageReferenceQueryParam'
 
-        - $ref: '#/components/parameters/limitQueryParam'
-        - $ref: '#/components/parameters/cursorQueryParam'
         - $ref: '#/components/parameters/Api-Version-Major'
       tags:
         - Port Call Service
       operationId: get-terminal-call
       summary: Retrieves a list of Terminal Calls
       description: |
-        Description...
+        Retrieves a list of **Terminal Calls** that match the specified filter criteria. It is **mandatory** to provide at least 1 of the following filters:
+          * `portCallID`
+          * `terminalCallID`
+          * `carrierServiceName`
+          * `carrierServiceCode`
+          * `universalServiceReference`
+          * `terminalCallReference`
+
+        The following filters can be used together with `carrierServiceName`, `carrierServiceCode` or `universalServiceReference` to narrow down the resultset even further:
```

```suggestion
        The following filters can also be used together with `carrierServiceName`, `carrierServiceCode` or `universalServiceReference` to narrow down the resultset even further:
```
The previous filters in the above section can also be combined already.. they can work together, I assume.